app/controllers/webhooks/websubs_controller.rb

```diff
@@ -17,7 +15,7 @@ def superfeedr_ingest
 
     # Rewrite default_news_items cache if any items.
     if items.count > 0
-      get_default_news_items(rewrite_cache: true)
+      NewsServices::get_default_news_items(rewrite_cache: true)
```

Comment:
I like this style better as it's more obvious that this is a call to a service object versus the other calls that route through a helper first which is non-standard.